pkg/apis/bundle/v1alpha1/bundle_types.go

```diff
@@ -69,22 +67,22 @@ type ComponentPackageSpec struct {
 	// `kind`, and `metadata`.
 	//
 	// This is essentially equivalent to the Kubernetes `Unstructured` type.
-	ClusterObjects []unstructured.Unstructured `json:"clusterObjects,omitempty"`
+	Objects []*unstructured.Unstructured `json:"objects,omitempty"`
```

Comment:
Were we going to use runtime.RawExtension (external v1alpha1 type) and runtime.Object (internal type) for unstructured data?

Reply:
Ultimately, I felt like runtime.RawExtension wasn't as good a fit as Unstructured. It's possible that the objects in the Bundle don't have a DeepCopy method (so don't fit runtime.Object) and are not the relevant K8S schema. Unstructured basically says: This should have K8S object and type meta, which is the currently how I view these things.